misc/config_tools/data/tgl-rvp/hybrid.xml

```diff
@@ -20,6 +20,7 @@
             <RDT_ENABLED>n</RDT_ENABLED>
             <CDP_ENABLED>n</CDP_ENABLED>
         </RDT>
+        <NVMX_ENABLED>n</NVMX_ENABLED>
```

Comment:
nested-virtualization is target enabled on tgl-hybird scenario first right, so should we set NVMX_enabled as 'y' by default?

Reply:
Nested VMX feature is still an POC feature. So, it won't be turn on by default.